flink/v1.16/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergFilesCommitter.java

```diff
@@ -69,6 +69,7 @@ class IcebergFilesCommitter extends AbstractStreamOperator<Void>
 
   private static final Logger LOG = LoggerFactory.getLogger(IcebergFilesCommitter.class);
   private static final String FLINK_JOB_ID = "flink.job-id";
+  private static final String OPERATOR_UNIQUE_ID = "flink.operator-unique-id";
```

Comment:
nit: would `flink.operator-id` be sufficient? it also matches the `job-id` style. `id` usually implies unique.

Reply:
similarly, the constant name can be `FLINK_OPERATOR_ID`